app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorModule.kt

```diff
@@ -15,4 +18,9 @@ class SyncIndicatorModule {
     val typeConverter = LastSyncedState.RxPreferenceConverter(moshi)
     return rxSharedPreferences.getObject("last_frequent_sync_result_v1", LastSyncedState(), typeConverter)
   }
+
+  @Provides
+  fun syncIndicatorModule(): Single<SyncIndicatorConfig> {
```

Comment:
Did you copy the class name as this functions name? 😄

Reply:
No I did not. I will rename it if that's what you mean though.

Reply:
Done